spec/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner_spec.rb

```diff
@@ -140,6 +138,42 @@
     end
   end
 
+  describe '#post_ansible_run' do
+    let(:options) { {:inventory => 'inv1', :job_template_ref => 'jt1'} }
+
+    context 'playbook runs successfully' do
+      it 'removes temporary inventory and job template and finishes the job' do
+        expect(subject).to receive(:delete_inventory)
+        expect(subject).to receive(:delete_job_template)
+        subject.post_ansible_run('Playbook ran successfully', 'ok')
+        expect(subject.state).to eq('finished')
```

Comment:
You can use `expect(subject).to have_attributes(:state => "finished", :status => "ok")` in each of these to simplify the number of expectations.